### ISSUE TITLE
can't show Chinese character

### ISSUE THREAD

**Issue by kingmpw2015** [jjuback/gc-excelviewer#135]
great job, but the Chinese character is not rendered correctly, just UTF-8 code.

```&#21734;&#65292;&#29616;&#22312;&#36824;&#19981;&#34892;&#65292;&#22240;&#20026;&#26368;&#36817;&#36164;&#37329;&#21608;&#36716;&#12290;***&#36825;&#20010;&#30340;&#35805;&#35831;&#20320;&#24819;&#21150;&#27861;&#20811;&#26381;&#19968;&#19979;&#65292;&#27605;&#31455;&#27424;&#27454;&#24050;&#32463;&#36926;&#26399;&#20102;&#23545;&#21543;&#12290;____intent:&#21015;&#20030;&#22256;&#38590;_model:3000034_round:2_score:0.57&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9608;&#9610;
&#21571;&#65292;&#23545;&#36825;&#20010;&#25105;&#30693;&#36947;&#12290;***&#22909;&#30340;&#65292;&#37027;&#25105;&#36825;&#36793;&#21518;&#32493;&#20063;&#20250;&#20851;&#27880;&#20320;&#30340;&#36824;&#27454;&#24773;&#20917;&#65292;&#20877;&#35265;&#12290;____intent:&#36824;&#27454;&#26102;&#38388;&lt;=3&#22825;_model:3000034_round:3_score:0.98 ```

**Comment by jjuback** [comment]
The extension does support Asian languages:

<img width="290" alt="JapaneseSample" src="https://user-images.githubusercontent.com/18399724/87976263-d2e1a900-ca9a-11ea-8c56-668a70c69b71.png">

If you are expecting it to convert HTML Unicode sequences to Chinese characters, it won't do that. If you attach a sample data file I can take a look.

**Comment by BurgessX** [comment]
I have the same problem. And I find that only .xlsx files do not support Chinese charaters, but .csv files do.
The file is attached here:
[TopMovieList.xlsx](https://github.com/jjuback/gc-excelviewer/files/7097808/TopMovieList.xlsx)
Hope the bug(maybe) can be solved.

**Comment by jjuback** [comment]
@kingmpw2015 @BurgessX This is fixed in version 3.0.44. The problem was only with .xlsx files.
